language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -582,6 +608,13 @@ test('party API', async () => {
 
 });
 
+
+test('user API', async () => {
+  const ledger = new Ledger({token: USER_TOKEN, httpBaseUrl: httpBaseUrl()});
+  const user = await ledger.getUser()
```

Comment:
this is still failing because the user doesn’t exist. I suspect the grpcurl might be failing. I recommend running this test locally until it succeeds.

Reply:
Quite confusing, I thought it does succeed locally. Will take another look

Reply:
Figured the problem simply was that grpcurl was not in the PATH such that it couldn't be even called.